applicationconfig/client/src/main/java/com/azure/applicationconfig/AzConfigClient.java

```diff
@@ -63,39 +64,38 @@
      * @return an instance of AzConfigClient
      */
     public static AzConfigClient create(String connectionString) {
-        return create(connectionString, new PipelineOptions());
+        return create(
+                connectionString,
+                HttpLogDetailLevel.BASIC);
     }
 
     /**
      * Create a new instance of AzConfigClient with pipeline options that uses connectionString for authentication.
      * @param connectionString connection string in the format "Endpoint=_endpoint_;Id=_id_;Secret=_secret_"
-     * @param pipelineOptions pipeline options
+     * @param logLevel Amount of detail to log for requests sent and responses received
      * @return an instance of AzConfigClient
      */
-    public static AzConfigClient create(String connectionString, PipelineOptions pipelineOptions) {
-        AzConfigCredentials credentials = AzConfigCredentials.parseConnectionString(connectionString);
-        return new AzConfigClient(credentials, pipelineOptions);
+    public static AzConfigClient create(String connectionString, HttpLogDetailLevel logLevel) {
```

Comment:
Are you still working through my comments? One of them was around String as the type for connection string.

Reply:
Ahh yes, I remember ConnectionDetails rather than just a string. I'll go and revise that. :) Sorry! That PR has so many comments, I should manually start resolving some as they get checked off.

Reply:
Created issue https://github.com/Azure/azure-sdk-for-java/issues/3080 to investigate.